reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -39,15 +62,28 @@ class LocalTime(val hour: Int, val minute: Int) : Serializable {
         }
 
         private fun create(hour: Int, minute: Int): LocalTime {
-            timeChecker(hour, minute)
+            validateHour(hour)
+            validateMinute(minute)
             return LocalTime(hour, minute)
         }
 
-        private fun timeChecker(hour: Int, minute: Int) {
-            if (hour > MAX_HOUR || hour < MIN_HOUR)
-                throw Exception("Incorrect hour")
-            if (minute < MIN_MINUTE || minute > MAX_MINUTE)
-                throw Exception("Incorrect minutes")
+        internal fun isTimeWithinMinMaxValue(
+            time: LocalTime,
+            minTime: LocalTime,
+            maxTime: LocalTime
+        ): Boolean = if (time.hour == minTime.hour && time.hour == maxTime.hour) {
+                isWithinMinMaxRange(time.minute, minTime.minute, maxTime.minute)
+            } else {
+                isWithinMinMaxRange(time.hour, minTime.hour, maxTime.hour)
+                        && isWithinMinMaxRange(time.minute, minTime.minute, maxTime.minute)
+            }
```

Comment:
Does this condition really true?
Assume minTime - 12.10, maxTime - 13.10
What will be the function result for 13.00. 
In real time world it belongs to the range of 12.10 .. 13.10.